packages/webapp/src/components/RepeatCropPlan/Confirmation.jsx

```diff
@@ -0,0 +1,218 @@
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation, Trans } from 'react-i18next';
+import { Info, Main } from '../Typography';
+import Form from '../Form';
+import Button from '../Form/Button';
+import MultiStepPageTitle from '../PageTitle/MultiStepPageTitle';
+import { getTextAndOccurrences, RRULEDAYS } from './utils';
+import {
+  CROP_PLAN_NAME,
+  PLAN_START_DATE,
+  REPEAT_FREQUENCY,
+  REPEAT_INTERVAL,
+  DAYS_OF_WEEK,
+  MONTH_REPEAT_ON,
+  FINISH,
+  AFTER_OCCURRENCES,
+  FINISH_ON_DATE,
+} from './constants';
+import styles from './styles.module.scss';
+
+/**
+ * Function that formats date to include day of week.
+ * @param {string} date Date to format. ex. '2023-12-31'
+ * @param {string} language User's preferred language. ex. 'en', 'es', 'fr', 'pt'
+ * @returns {string} Formatted date. ex. 'Sunday, December 31, 2023'
+ */
+const getDate = (date, language = 'en') => {
+  const [year, month, day] = date.split('-');
+  return new Intl.DateTimeFormat(language, { dateStyle: 'full' }).format(
+    new Date(year, month - 1, day),
+  );
+};
+
+function PureRepeatCropPlanConfirmation({
+  useHookFormPersist,
+  persistedFormData,
+  onGoBack,
+  onSubmit,
+  tasks,
+}) {
+  const { t } = useTranslation();
+  const { historyCancel } = useHookFormPersist();
+
+  const {
+    planName,
+    beginning,
+    finishingText,
+    numberOfPlans,
+    numberOfTasks,
+    willRepeatText,
+    occurrences,
+  } = useMemo(() => {
+    const planName = persistedFormData[CROP_PLAN_NAME];
+    const planStartDate = persistedFormData[PLAN_START_DATE];
+    const repeatFrequencyNumber = persistedFormData[REPEAT_FREQUENCY];
+    const finish = persistedFormData[FINISH];
+
+    const daysOfWeek = persistedFormData[DAYS_OF_WEEK];
+    const monthRepeatOn = persistedFormData[MONTH_REPEAT_ON] || {};
+    const finishOnDate = persistedFormData[FINISH_ON_DATE];
+    const afterOccurrences = persistedFormData[AFTER_OCCURRENCES];
+    const repeatInterval = persistedFormData[REPEAT_INTERVAL];
+
+    let finishingText = '';
+    if (finish === 'on') {
+      finishingText = t('REPEAT_PLAN.CONFIRMATION.ON_DATE', {
+        date: getDate(finishOnDate),
+      });
+    } else if (finish === 'after') {
+      finishingText = (
+        <Trans i18nKey={'REPEAT_PLAN.CONFIRMATION.AFTER_REPETITIONS'}>
+          after <b>{{ number: afterOccurrences }}</b> occurrences
+        </Trans>
+      );
+    }
+
+    const { text: willRepeatText, occurrences } = getTextAndOccurrences(
```

Comment:
Is the `willRepeatText` missing a reference to the RRule translations?

![Screenshot 2023-07-27 at 8 46 18 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/82d5a072-6e55-493d-9f0f-520eee1f9e0f)

Reply:
Thank you so much for catching this!!!!!